### PR TITLE
[PT-69]-LoginModal

### PR DESCRIPTION
**Pull request by ItsFaizan**
# [PT-69](https://0xbase.atlassian.net/browse/PT-69?atlOrigin=eyJpIjoiOThlYTY1OGRmMjBmNDk3Mzk3MzMyNDMxNGZhYjE0ZTAiLCJwIjoiaiJ9) - LoginModal

Login Modal created and divider created as a custom component

[PT-69]: https://0xbase.atlassian.net/browse/PT-69?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/container/LoginDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const LoginDialog: FC<LoginProps> = ( ) => {
+
+    return (
+        <Modal title="Login" openModalButton='Login'>
+
+            <TextField fullWidth label="Email" margin="normal" />
+            <TextField fullWidth label="Password" type="password" margin="normal" />
+            
+            <Typography sx={{ textAlign: 'right', mb: 2 }}>
+                Don't have an account? <Typography component="a" href='#' >Signup</Typography>
```
Comment (fuddin95) [inline src/container/LoginDialog.tsx]: Now that we have `Signup Modal` we can show that view once a user clicks Signup

File: src/app/(home)/page.tsx [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+
 
 export default function Home() {
   return (
     <main >
       <h1>Landing Home Page</h1>
+      <LoginDialog onSignupClick={function (): void {
+        throw new Error("Function not implemented.")
+      } } onConnectWalletClick={function (): void {
+        throw new Error("Function not implemented.")
```
Comment (fuddin95) [inline src/app/(home)/page.tsx:13]: 1. Your functions  are not beign used in the LoginDialog.
2. Lets avoid inline functions.
3. Lets not throw Errors rather think of handling errors gracefully

**Review by fuddin95 — CHANGES_REQUESTED** [review]

File: src/components/divider/CustomDivider.tsx
Comment (fuddin95) [inline src/components/divider/CustomDivider.tsx:1]: Generally the best practice is to keep the folder and component name same so you can just name the 
- folder `divider`
- component `Divider`

File: src/app/(home)/page.tsx (hunk 2 above)
Comment (ItsFaizan) [reply src/app/(home)/page.tsx:13]: Removed these functions

**Pull request merged**